packages/react/src/components/Rating/Rating.types.ts

```diff
@@ -110,11 +110,16 @@ export interface IRatingProps extends React.HTMLAttributes<HTMLDivElement>, Reac
    */
   readOnly?: boolean;
 
-  /*
-   * Optional callback to set the aria-label for rating control.
+  /**
+   * Optional callback to set the aria-label for rating control in readOnly mode
    */
   getAriaLabel?: (rating: number, max: number) => string;
 
+  /**
+   * Optional aria-label for rating control
+   */
+  ariaLabel?: string;
```

Comment:
Would be nice to have remarks clarifying how `ariaLabel` and `getAriaLabel` interact. Which one is preferred?